.eslintrc.yaml

```diff
@@ -1,5 +1,6 @@
----
-extends: "@commercetools-frontend/eslint-config-mc-app"
+extends: '@commercetools-frontend/eslint-config-mc-app'
 settings:
   react:
     version: '^16.0'
+  import/ignore:
+    - test-utils.js$
```

Comment:
We do `export * from 'react-testing-library'` in `test-utils.js`, which confuses the `import/named` rule. It complains when doing `import { fireEvent } from '<xy>/test-utils'` as `fireEvent` is one of the forwarded exports.

To avoid a warning in every consumer, we simply ignore `test-utils`. This is something all consumers of `application-kit` (more specifically, `@commercetools-frontend/application-shell/src/test-utils`) which have `import/named` enabled need to add to their ESLint setup as well.

Reply:
Then every single test which imports anything other than `render` or `renderWithRedux` from `test-utils` would have to use `eslint-ignore`. 

Aside from that, which upside do you see in using `eslint-ignore` over disabling the rule only for that import specifically?

Reply:
That's a valid point. I also like the honesty of having it centralised in an ESLint config. We have spread ignoring to export a default export throughout the app locally. Maybe we should also add that here given we don't seem to mind.

Reply:
Yeah, we can do another PR to improve our ESLint rules. I'd want to disable [arrow-body-style](https://eslint.org/docs/rules/arrow-body-style) and [prefer-default-export](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md). They both don't add much value and I find myself often changing back-and-forth between `return` and no return when debugging.